### PR TITLE
Added Legend of Zelda: Ocarina of Time

### PR DESCRIPTION
**Pull request by Mailaender**
via https://www.youtube.com/watch?v=kbc7IzgTvuU

### CODE REVIEW

File: games/z.yaml [hunk 1]
```diff
@@ -140,6 +140,21 @@
   - 'https://www.zeldaclassic.com/wp-content/uploads/2016/12/ss_hero_dreams.png'
   - 'https://www.zeldaclassic.com/wp-content/uploads/2016/12/ss_quest744.png'
 
+- name: 'The Legend of Zelda: Ocarina of Time'
```
Comment (cxong) [inline games/z.yaml:143]: this should be in `l.yaml`

File: originals/z.yaml [hunk 2]
```diff
@@ -15,6 +15,14 @@
   names:
   - Virus
 
+- name: 'The Legend of Zelda: Ocarina of Time'
```
Comment (cxong) [inline originals/z.yaml:18]: same here

File: originals/z.yaml [hunk 3]
```diff
@@ -15,6 +15,14 @@
...
+  external:
+    wikipedia: 'The Legend of Zelda: Ocarina of Time'
+  meta:
+    genre:
+    - Action
+    - Adventure
```
Comment (cxong) [inline originals/z.yaml:24]:
```suggestion
    - Adventure
    theme:
    - Fantasy
```

File: games/z.yaml [hunk 4]
```diff
@@ -140,6 +140,21 @@
...
+  originals:
+  - 'The Legend of Zelda: Ocarina of Time'
+  type: remake
+  repo: 'https://github.com/zeldaret/oot'
+  url: 'https://zelda64.dev/'
+  development: active
+  status: playable
+  license:
+  - Custom
```
Comment (cxong) [inline games/z.yaml:152]: should this be As-is? I can't find a license in the repo

File: games/z.yaml [hunk 5]
```diff
@@ -140,6 +140,21 @@
...
+  content: commercial
+  lang:
+  - C
+  updated: '2021-12-12'
```
Comment (cxong) [inline games/z.yaml:156]: let's add some images/video; maybe https://www.youtube.com/watch?v=kbc7IzgTvuU will do if there isn't an official one

**Review by cxong — CHANGES_REQUESTED** [review]

File: games/z.yaml (hunk 5 above)
Comment (Mailaender) [reply games/z.yaml]: I added an image. There is nothing from the developers.

File: games/z.yaml (hunk 4 above)
Comment (Mailaender) [reply games/z.yaml]: Yes, it is just like re3 without a license.

File: games/z.yaml (hunk 1 above)
Comment (Mailaender) [reply games/z.yaml]: okay

**Review by cxong — APPROVED** [review]

**Pull request merged**